app/services/distribution_update_service.rb

```diff
@@ -1,17 +1,13 @@
-class DistributionUpdateService
-  attr_reader :distribution, :error
-
+class DistributionUpdateService < DistributionService
   def initialize(old_distribution, new_distribution_params)
     @distribution = old_distribution
     @params = new_distribution_params
-    @organization = @distribution.organization
-    @error = nil
   end
 
   # FIXME: This doesn't allow for the storage location to be changed.
```

Comment:
Off topic, but @armahillo this fixme can be changed, right?

Reply:
Possibly. I would need to dive back in to look. I forget the context :grimacing:

Reply:
Ah, now I remember the context:

Could probably write a test for this, but if you change the `storage_location_id` of the distribution, that should be reflected in this (or else we shouldn't allow that to be changed....but I think that's a requirement?). If the storage location is changed, additional stuff will need to happen. But this would be the right place to do it, I think!